handsontable/src/plugins/customBorders/customBorders.js

```diff
@@ -636,8 +657,9 @@ export class CustomBorders extends BasePlugin {
    * Check if an border already exists in the customSelections, and if true call toggleHiddenClass method.
    *
    * @private
-   * @param {object} border Object with `row` and `col`, `left`, `right`, `top` and `bottom`, `id` and `border` ({Object} with `color`, `width` and `cornerVisible` property) properties.
-   * @param {string} place Coordinate where add/remove border - `top`, `bottom`, `left`, `right` and `noBorders`.
+   * @param {object} border Object with `row` and `col`, `start`, `end`, `top` and `bottom`, `id` and
```

Comment:
same as https://github.com/handsontable/handsontable/pull/9144/files#r793013086

Reply:
Thanks for spotting that but I think that kind of improvement I would leave for a separate issue. Especially that the border object in the different lifecycle of the plugin, different methods can have a different shape.

Reply:
as you want.